sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -240,4 +250,531 @@ describe("AssessmentController(e2e)-workflowWrapUp", () => {
       assessImpactedApplicationReassessmentNeededMock,
     ).not.toHaveBeenCalled();
   });
+
+  it("Should populate the previousDateChangedReportedAssessment when the e-Cert was generated for the 'Original Assessment' and offering dates changed as a result of the reassessment.", async () => {
+    // Arrange
+    const savedApplication = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      undefined,
+      {
+        applicationStatus: ApplicationStatus.Completed,
+        currentAssessmentInitialValues: {
+          assessmentDate: now,
+          studentAssessmentStatus: StudentAssessmentStatus.Completed,
+        },
+        createSecondDisbursement: false,
+        firstDisbursementInitialValues: {
+          coeStatus: COEStatus.completed,
+          disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+        },
+      },
+    );
+    const originalAssessmentId = savedApplication.currentAssessment.id;
+    const differentStudyStartDate = new Date(
+      savedApplication.currentAssessment.offering.studyStartDate,
+    );
+    differentStudyStartDate.setDate(differentStudyStartDate.getDate() + 1);
+    await saveOfferingAndAssessment(savedApplication, now, auditUser, {
+      studyStartDate: getISODateOnlyString(differentStudyStartDate),
+    });
+    savedApplication.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.InProgress;
+    await db.studentAssessment.save(savedApplication.currentAssessment);
+    const workflowData = {
+      studentData: {
+        dependantStatus: "independant",
+      },
+    } as WorkflowData;
```

Comment:
I know there were some tests with this already but I did a test here and this is just enough:
`const workflowData = {} as WorkflowData;`
Could you please replace it all across this file? I think that would be better for reading it as the workflow data is not important here.

Reply:
Or even better:
```
    // Act
    const result = await assessmentController.workflowWrapUp(
      createFakeWorkflowWrapUpPayload(
        savedApplication.currentAssessment.id,
        {} as WorkflowData,
      ),
    );
```